src/custom/constants/index.ts

```diff
@@ -1,22 +1,21 @@
 import { ChainId } from '@uniswap/sdk'
+import { GPv2Settlement, GPv2AllowanceManager } from '@gnosis.pm/gp-v2-contracts/networks.json'
 
 // TODO: Load app Id
 export const APP_ID = 0
 
-// TODO: The addresses will be exported as plain JSON file in the future
-
 // reexport all Uniswap constants everything
 export * from '@src/constants/index'
 
 // TODO: When contracts are deployed, we can load this from the NPM package
 export const GP_SETTLEMENT_CONTRACT_ADDRESS: Partial<Record<ChainId, string>> = {
-  [ChainId.MAINNET]: '0x8cEe82E709e770733180Ef222a8989B923fedE01', // TODO: No yet.
-  [ChainId.RINKEBY]: '0x828229A8432A89B8624B6AF91eC0BB65b9517156'
-  // TODO: extend chainId for adding xDAI
+  // [ChainId.MAINNET]: GPv2Settlement[1].address, // Not yet deployed
+  [ChainId.RINKEBY]: GPv2Settlement[4].address
```

Comment:
is `ChainId.Rinkeby` enum a number ? why not use that in the GPv2Settlement[ChainId.Rinkeby]` otherwise if its  astring or whatever ignore me

Reply:
No, u right!

I was thinking I had to use string at the begining, and actually at the beging did `ChainId.RINKEBY.toString()`, then I did a reduce, then I thought this is overkilling and I added a "4", then dima said 4 is enough!  I went in a big circle